Sources/Web5/Crypto/JOSE/JWT.swift

```diff
@@ -90,4 +91,64 @@ public struct JWT {
             )
         )
     }
+
+    public struct ParsedJWT {
+        let header: JWS.Header
+        let payload: [String: AnyCodable]
+
+        public init(
+            header: JWS.Header,
+            payload: [String: AnyCodable]
+        ) {
+            self.header = header
+            self.payload = payload
+        }
+    }
+
+    public static func parse(jwtString: String) throws -> ParsedJWT {
```

Comment:
just fyi as parse is getting renamed in 'decode' I think :

lots of new decoding logic here for making sure jwt is valid vcjwt
https://github.com/TBD54566975/web5-js/pull/421

and when we get to the verify part of the vcJwt the plan is to split up verify into lots of parts with verify options - https://github.com/TBD54566975/web5-js/issues/425